BabbleApp/babbleapp.py

```diff
@@ -217,59 +248,50 @@ async def async_main():
     if config.settings.gui_ROSC:
         osc_receiver = VRChatOSCReceiver(cancellation_event, config, cams)
         osc_receiver_thread = threading.Thread(target=osc_receiver.run)
-        osc_receiver_thread.start()
+        thread_manager.add_thread(osc_receiver_thread)
         ROSC = True
 
     # Create the window
     window = sg.Window(
-        f"{appversion}", layout, icon="Images/logo.ico", background_color=bg_color_clear
+        f"{AppConstants.VERSION}", layout, icon="Images/logo.ico", background_color=bg_color_clear
```

Comment:
(This should have been done already lol)

Please use `os.path.join()` for path resolution

```suggestion
        f"{AppConstants.VERSION}", layout, icon=os.path.join('Images', 'logo.ico'), background_color=bg_color_clear
```